src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -703,6 +703,19 @@ public static class WebView
 #endif
 		}
 
+		public static class WebView2
+		{
+#if __IOS__
+			/// <summary>
+			/// Sets wether the <see cref="WebView2"/> object is inspectable or not.
+			/// </summary>
+			/// <remarks>
+			/// On iOS and Catalyst this means that developers can use the Safari Web Developers tools to debug apps with <see cref="WebView2"/>
+			/// </remarks>
+			public static bool IsInspectable { get; set; };
```

Comment:
Please add docs for the new flag as well

Reply:
Working on that this morning.

Reply:
added :)